happy-hackers/Ming_MagentoTask
# create category in store front

### Description
Store Front Category Layer Navigation


1. Create a layer navigation in custom page
2. Using default category

### Manual testing scenarios

1. Go to custom page:local.magento/layerednavigation/index
2. Check the layer navigation left side

### Contribution checklist
